example/android/app/build.gradle

```diff
@@ -1,10 +1,10 @@
 apply plugin: 'com.android.application'
 
 android {
-    namespace "com.cursorinsight.trap.capacitor_example"
```

Comment:
Is the underscore the usual character to separate multiple words?  Can we have a dash instead?

Reply:
I've switched to using only separate words, as that is recommended by the Kotlin style guide (https://developer.android.com/kotlin/style-guide#package_names)